pgmpy/estimators/GES.py

```diff
@@ -55,6 +55,20 @@ def __init__(self, data, use_cache=True, **kwargs):
 
         super(GES, self).__init__(data=data, **kwargs)
 
+    def Test_type(self):#Ideally I would run this in over to choose the proper test depending on the kind of data.
```

Comment:
I would also suggest removing the comment here and writing a docstring instead. You can find examples of docstring format in most of the other methods.